core/arch/arm/mm/tee_pager.c

```diff
@@ -770,4 +809,38 @@ void tee_pager_add_pages(vaddr_t vaddr, size_t npages, bool unmap)
 		core_tlb_maintenance(TLBINV_UNIFIEDTLB, 0);
 	}
 }
+
+void tee_pager_release_zi(vaddr_t vaddr, size_t size)
+{
+	bool unmaped = false;
+	uint32_t exceptions = thread_mask_exceptions(THREAD_EXCP_ALL);
```

Comment:
Why are we masking exceptions in this function but not in the one below or vice versa?